web/beacon-app/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,138 @@
+import email from '/src/assets/icons/email.png';
+import github from '/src/assets/icons/github.png';
+import linkedin from '/src/assets/icons/linkedin.png';
+import twitter from '/src/assets/icons/twitter.png';
+import otter from '/src/assets/images/footer-otter.png';
+
+function Footer() {
+  return (
+    <footer className="bg-footer bg-cover bg-no-repeat">
+      <div className="pt-64 2xl:pt-80">
```

Comment:
As the screen width increases, content in the footer goes above the background image. `2xl:pt-80` is a small hack to prevent this from happening. While updating the website's footer, I'll look into a better way of handing this.